src/coffee/clients.coffee

```diff
@@ -212,9 +212,14 @@ class Clients
       scheme: 'instacast://'
       icon: 'osx/instacast.png'
     },
+    {
+      title: 'Podcasts'
```

Comment:
According to the marketing headline of Apple Podcasts (😅), the app only works on Catalina:
> Use the Apple Podcasts app on your iPhone, iPad, and iPod touch with the latest version of iOS or iPadOS, on your Mac with macOS Catalina, or on your Apple TV.

(https://support.apple.com/en-us/HT201859)

It probably shouldn't be listed for lower versions on macOS (if we figure out how to reliably detect Catalina-only)

Reply:
Thanks for finding this! Removed it from the clients list so that pre-Catalina only gets iTunes (with the legacy itpc:// protocol) and Catalina gets Podcasts.
Thinking about it further I think this PR also needs a way to detect post-Catalina, e.g. version 10.16 and above. Not sure if that's something we can do with a regex :/